src/sentry/static/sentry/less/includes/bootstrap.less

```diff
@@ -3,21 +3,17 @@
 @import './bootstrap/buttons.less';
 @import './bootstrap/button-groups.less';
 @import './bootstrap/dropdowns.less';
-@import './bootstrap/normalize.less';
 @import './bootstrap/navs.less';
 @import './bootstrap/print.less';
 @import './bootstrap/type.less';
-@import './bootstrap/code.less';
 @import './bootstrap/tables.less';
 @import './bootstrap/panels.less';
 @import './bootstrap/forms.less';
-@import './bootstrap/scaffolding.less';
-@import './bootstrap/grid.less';
 @import './bootstrap/list-group.less';
 @import './bootstrap/tooltip.less';
 @import './bootstrap/modals.less';
 @import './bootstrap/utilities.less';
-@import './bootstrap/responsive-utilities.less';
```

Comment:
Here I removed as much of the Bootstrap styles as I could and moved them into the `base.less` file